rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/execution/MainExecutionContext.kt

```diff
@@ -116,7 +132,15 @@ data class Context(
     val memorySliceMgr: MemorySliceMgr? = null,
     val programStack: Stack<RpgProgram> = Stack<RpgProgram>(),
     val systemInterface: SystemInterface,
-    var executionProgramName: String? = null
+    var executionProgramName: String? = null,
+    val dbFileFactory: DBFileFactory? = configuration.reloadConfig?.let {
```

Comment:
Also this part is a temporary solution. In my mind, all metadatas are already registered in Reload because someone has already loaded the tables into Reload and recorded the metadata. Jariko cannot open a table if this table is not already registered in Reload by someone else. For now since this someone is not clear who he is, I record the necessary metadata in this wrong way.